src/plugins/data_source/server/saved_objects/data_source_saved_objects_client_wrapper.ts

```diff
@@ -476,4 +490,15 @@ export class DataSourceSavedObjectsClientWrapper {
       },
     };
   }
+
+  private async getAuthenticationMethodFromRegistry(type: string) {
+    const authMethod = (await this.authRegistryPromise).getAuthenticationMethod(type);
+    return authMethod;
+  }
+
+  private async isAuthTypeAvailableInRegistry(type: string): Promise<boolean> {
+    const authMethod = await this.getAuthenticationMethodFromRegistry(type);
+    return authMethod !== undefined;
+    return false;
```

Comment:
looks like we have 2 return statements here, remove line 502?

Reply:
done. removed.